doc/docker.md

```diff
@@ -66,10 +66,12 @@ thing.
 There are several docker-compose overrides you can use to customize your docker-compose
 environment. These can be found in `docker/dev/`. Currently these overrides support:
 
-- assigning random ports to rails and solr
-- sharing a ssh-agent with the app service so you can do capistrano deploys
-- using unison for faster performance on OS X
-- external mysql server
+- `docker-compose-external-mysql.yml`: external mysql server
+- `docker-compose-random-ports.yml`: assigning random ports to rails and solr
+- `docker-compose-solr-test.yml`: run an additional solr docker instance for user with rspec and cucumber testing
```

Comment:
Since running tests that require solr is something every developer should be doing. I think this can be moved into the main docker-compose file instead of being an override.